src/CLI/clitree/cli-xml/ipv4.xml

```diff
@@ -79,6 +79,42 @@ limitations under the License.
 	    python $SONIC_CLI_ROOT/sonic-cli-if.py delete_openconfig_if_ip_interfaces_interface_subinterfaces_subinterface_ipv4_addresses_address_config_prefix_length ${iface} ${addr}
     </ACTION>
     </COMMAND>
-  </VIEW>
+</VIEW>
+
+<VIEW name="configure-if-mgmt-view">
+
+    <COMMAND
+        name="ip"
+        help="Interface Internet Protocol config commands"
+    />
+    <COMMAND
+        name="no ip"
+	help="Interface Internet Protocol config commands"
```

Comment:
Please check the indentation in this file

Reply:
done